_posts/2018-11-16-Episode-7-GeoViews.md

```diff
@@ -0,0 +1,96 @@
+While the plotting space is rather crowded, the geographic plotting space is not as crowded.  There are projects such as Cartopy supports plotting using Matplotlib, which Geoviews has been built on.  Ipyleaflet also allows plotting on tile sources, as well as offers some interactivity.
+Plotly would be the last project to compare to GeoViews, and it provides some integration for geo support.
+
+GeoViews is a component of the broader initiative to make it easier to deal with very different datasets in the pydata ecosystem.  Mainly it consists of two components: subclassed HoloViews elements with Cartopy projections, and operations to project different types of data.  There is also the data backend which enables GeoViews to natively understand the data which includes libraries like Xarray, Iris, Geopandas and so on.  Matplotlib forms the basis for the plotting and rendering components.  Bokeh support has been the focus most recently, and this component allows true interactivity, e.g. cross-filtering, drawing tools etc.
+
+Before PyViz was a thing, there was a small team at Anaconda and in collaboration with MetOffice, the GeoViews project was started.  The MetOffice is the short name for the meteorological office which is a government department in the UK and conducts weather forecasting as well as other climatology research.  From that point on, Martin has taken on the maintenence.
+
+Clients have always been a big support for this project moving forward.  More broadly speaking, the users have been fairly scattered, using it for personal projects.  Another big user group comes from other government agencies in the U.S. like the USGS., NASA, and the U.S. Army Core of Engineers.  We have also recently begun working with the PanGeo research project, which is a collaborative effort run by various geographic and oceanographic organizations who want to utilize cloud computing. So at this point, the PyViz team (mostly Phillip) funded through consulting projects with industry and government clients.  Right now this project is trying to become more welcoming, and the effort is to get more people to join the project and increase the diversity of the team and leadership. 
+
+
+### Demo
+
+[Basics of wrapping gridded/tabular/geometry data with projection](http://geoviews.org/user_guide/Projections.html)
+[hvPlot as a simpler entry point](https://hvplot.pyviz.org/user_guide/Geographic_Data.html)
+[Various examples and dashboards](http://geoviews.org/gallery/index.html
+[Glacier Crossfilter App](https://anaconda.org/philippjfr/oggm_glacier-raw/notebook)
+[EarthSim demos, e.g. drawing tools etc.](http://earthsim.pyviz.org/topics/GrabCut.html)
```

Comment:
Also, all of the above demo lines should probably start with a `* ` so that they appear as a bullet pointed list. For example, 

```
* [Glacier Crossfilter App](https://anaconda.org/philippjfr/oggm_glacier-raw/notebook)
* [EarthSim demos, e.g. drawing tools etc.](http://earthsim.pyviz.org/topics/GrabCut.html)
```

would become:

* [Glacier Crossfilter App](https://anaconda.org/philippjfr/oggm_glacier-raw/notebook)
* [EarthSim demos, e.g. drawing tools etc.](http://earthsim.pyviz.org/topics/GrabCut.html)